packages/snaps-controllers/src/interface/SnapInterfaceController.ts

```diff
@@ -272,6 +272,7 @@ export class SnapInterfaceController extends BaseController<
     );
 
     await this.#triggerPhishingListUpdate();
+    // TODO: Expose configuration for allowedProtocols in validateJsxLinks
```

Comment:
This change isn't very impactful until this is done, since all UI is converted to JSX as of recently.